Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -651,7 +651,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     bool shouldEnableLockdownMode() const;
 
     bool hasSameGPUAndNetworkProcessPreferencesAs(const API::PageConfiguration&) const;
-    bool hasSameGPUAndNetworkProcessPreferencesAs(const WebPageProxy& page) const { return hasSameGPUAndNetworkProcessPreferencesAs(Ref { page }->configuration()); }
+    bool hasSameGPUAndNetworkProcessPreferencesAs(const WebPageProxy&) const;
```

Comment:
Why is this no longer inline?

Reply:
Because the static analyzer says that we hold a Ref to the configuration on the stack before passing it into the function. So we need to use `protectedConfiguration`. For that we need to know about APIPageConfiguration, which the header WebPageProxy.h does not. So to avoid an include, I moved this to the cpp file.